runbotics/runbotics-scheduler/src/queue/queue.service.ts

```diff
@@ -179,8 +179,14 @@ export class QueueService implements OnModuleInit {
         this.logger.log('Cleared staled job(s)');
     }
 
-    private async handleAttendedProcess(process: IProcess, input?: ProcessInput) {
-        if (!process.isAttended) return;
+    async handleAttendedProcess(process: IProcess, input?: ProcessInput) {
+        if (!process.isAttended && !input.variables) return;
+
+        if (!process.isAttended && input.variables) {
+            const err = 'You have provided variables for non attended process';
+            this.logger.error(`Failed to add new instant job for process "${process.name}". ${err}`);
+            throw new BadRequestException(err);
+        }
```

Comment:
imo it should not be an error. the process does not force you to provide variables if its not attended but having those should not throw an error - let it pass